src/parser/Triples.h

```diff
@@ -0,0 +1,79 @@
+//  Copyright 2024, University of Freiburg,
+//  Chair of Algorithms and Data Structures.
+//  Author: Julian Mundhahs (mundhahj@informatik.uni-freiburg.de)
```

Comment:
Call this File `SparqlTriple.h` (closer to the class names) and add me as an author (I think most of the actual code is unchanged).